src/FileMigrationHelper.php

```diff
@@ -34,6 +35,21 @@ class FileMigrationHelper
      */
     private static $delete_invalid_files = true;
 
+    private static $dependencies = [
```

Comment:
This doesn't log on CLI by default - we need to cover *both* use cases

Reply:
Actually, three: Queuedjob (through CLI), direct CLI run, and web-based run. This shouldn't be down to the task or task helper to sort out, but unfortunately the toplevel `TaskRunner` isn't great (https://github.com/silverstripe/silverstripe-framework/issues/5542)